crates/cli/src/datadog_utils.rs

```diff
@@ -59,21 +59,7 @@ pub fn get_secrets_rules(use_staging: bool) -> Result<Vec<SecretRule>> {
     let api_response = serde_json::from_str::<StaticAnalysisSecretsAPIResponse>(response_text);
 
     match api_response {
-        Ok(d) => Ok(d
-            .data
-            .iter()
-            .map(|v| SecretRule {
-                id: v.id.clone(),
-                name: v.attributes.name.clone(),
-                description: v.attributes.description.clone(),
-                pattern: v.attributes.pattern.clone(),
-                default_included_keywords: v
-                    .attributes
-                    .default_included_keywords
-                    .clone()
-                    .unwrap_or_default(),
-            })
-            .collect()),
+        Ok(d) => Ok(d.data.iter().map(|v| v.clone().into()).collect()),
```

Comment:
just consume it, d isn't used elsewhere

```suggestion
        Ok(d) => Ok(d.data.into_iter().map(|v| v.into()).collect()),
```